### PR TITLE
Add few missing fields in Stack report response model

### PR DESCRIPTION
**Pull request by arajkumar**

### CODE REVIEW

File: v2/stack_analyses.yaml [hunk 1]
```diff
@@ -544,6 +546,11 @@ components:
       title: StackAnalysesResult
       type: object
       required:
+        - version
+        - started_at
+        - finished_at
```
Comment (dgpatelgit) [inline v2/stack_analyses.yaml:551]: Can we rename this as "ended_at", as we use this attributes in the backbone flow. We can unify the naming of attributes. Also started_at looks perfect with ended_at

File: v2/stack_analyses.yaml [hunk 2]
```diff
@@ -557,6 +564,21 @@ components:
           registered: '#/components/schemas/StackAnalysesResultForRegisteredUser'
           freetier: '#/components/schemas/StackAnalysesResultForFreeTier'
       properties:
+        version:
```
Comment (prashbnair) [inline v2/stack_analyses.yaml:567]: Why do we need this? Isnt it always V2

File: v2/stack_analyses.yaml [hunk 3]
```diff
@@ -557,6 +564,21 @@ components:
...
+          type: string
+          example: v2
+        started_at:
```
Comment (prashbnair) [inline v2/stack_analyses.yaml:570]: What is purpose of the started and ended time? Is it displayed in UI?

File: v2/stack_analyses.yaml [hunk 4]
```diff
@@ -557,6 +564,21 @@ components:
...
+          type: string
+          example: "2020-04-06 11:56:08.349405"
+        ended_at:
+          type: string
+          example: "2020-04-06 11:58:18.349405"
+        external_request_id:
```
Comment (prashbnair) [inline v2/stack_analyses.yaml:576]: Isnt this also part of the request?

File: v2/stack_analyses.yaml (hunk 3 above)
Comment (arajkumar) [reply v2/stack_analyses.yaml:570]: yep., there is no real need right time. @dgpatelgit , Can you elaborate why you need this?

File: v2/stack_analyses.yaml (hunk 4 above)
Comment (arajkumar) [reply v2/stack_analyses.yaml:576]: The place where you POST the request(vs extension) and the place where you GET the response is different. So it is good to have id as part of the response.

File: v2/stack_analyses.yaml (hunk 2 above)
Comment (arajkumar) [reply v2/stack_analyses.yaml]: removed it.

File: v2/stack_analyses.yaml (hunk 3 above)
Comment (arajkumar) [reply v2/stack_analyses.yaml]: Removed it

File: v2/stack_analyses.yaml (hunk 4 above)
Comment (arajkumar) [reply v2/stack_analyses.yaml]: Removed it

**Pull request merged**